lte/cloud/go/services/subscriberdb/digest_test.go

```diff
@@ -251,3 +252,193 @@ func TestGetDigestApnResourceAssocs(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotEqual(t, expected, digest)
 }
+
+func TestGetPerSubDigests(t *testing.T) {
+	lte_test_init.StartTestService(t)
+	configurator_test_init.StartTestService(t)
+
+	err := configurator.CreateNetwork(configurator.Network{ID: "n1"}, serdes.Network)
+	assert.NoError(t, err)
+	_, err = configurator.CreateEntity("n1", configurator.NetworkEntity{Type: orc8r.MagmadGatewayType, Key: "g1", PhysicalID: "hw1"}, serdes.Entity)
+	assert.NoError(t, err)
+	gw, err := configurator.CreateEntity("n1", configurator.NetworkEntity{Type: lte.CellularGatewayEntityType, Key: "g1"}, serdes.Entity)
+	assert.NoError(t, err)
+
+	// Initially there should only be the placeholder apn digest in the generated digest map
+	perSubDigests, err := subscriberdb.GetPerSubDigests("n1")
+	assert.NoError(t, err)
+	assert.Equal(t, []*lte_protos.SubscriberDigestByID{}, perSubDigests)
+
+	// Generate individual digests for each newly detected subscriber in the network
+	_, err = configurator.CreateEntities("n1", configurator.NetworkEntities{
```

Comment:
Nit here and elsewhere: move each arg to "their own" line -- so newline after opening `(` and before closing `)`, etc